modules/ROOT/pages/setup-payload-storage-API.adoc

```diff
@@ -1,29 +1,33 @@
 = Payload Storage API for Partner Manager
 
-Anypoint Partner Manager enables you to configure your own storage solution, such as database, Amazon S3, Azure Blob Storage, and so on to store transmission content, such as incoming and outgoing B2B message payloads.  
+Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) to store transmission content, such as incoming and outgoing B2B message payloads.
```

Comment:
```suggestion
Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) for transmission content, such as incoming and outgoing B2B message payloads.
```
"storage solution for transmission content" or other wording not as redundant as "storage solution to store transmission content" (a storage solution obviously stores...)

Reply:
And do we need both "transmission content" and "incoming and outgoing B2B message payloads ("messages" would do) here?
Look for ways to reduce verbage in the legacy content: it is overinflated.

Reply:
Removed "such as" section